features/install.feature

```diff
@@ -13,9 +13,9 @@ Feature: Installation
     Project: Test Project
     Default Locale: en (English)
     """
-      And help should not be displayed
-      And a file named "config/initializers/localeapp.rb" should exist
-      And the exit status should be 0
+    And help should not be displayed
+    And a file named "config/initializers/localeapp.rb" should exist
+    And the exit status should be 0
```

Comment:
I agree it may/should be fixed, and that we have some work to do on
conventions etc… However this specific change should be a new
branch/PR (not even sure we'll change scenarios alone just for
conventions, to avoid churn: I realize now that all of them increase
indent for `And` steps :/).

Same comment for other scenarios not related to this PR, just ensure
we don't indent `And` steps in new scenarios we add.